data/hai/hai reveal 4 middle.txt

```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	One of the Elders takes the time to address you as a witness, and confirm that threat was indeed real. Even so, though, that only seems to sway a small selection. One of the more stubborn Elders stands up. "The Unfettered have made serious attempts to push through at Wah Ki before, to no avail. More recently, they've been more trouble again, and again we have organized more than enough security to keep their disruption contained, even with the current human crisis. This is ridiculous, and the Unfettered would not dare, not while they are so dependent on our food and medical supplies these past years."`
-			`	"You are blind. Blind and old and foolish," snaps Teeneep. "If you will not even listen to this, then apparently I must now be the one to present a pathway to avoid war. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
-			`	Two different Elders respond simultaneously. The first says, "Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
-			`	That one is left searching the room for someone who might actually be listening halfway through his sentence, though, as the second points at Teeneep and loudly says, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
-			`	Sayari has to raise her voice above the din and disentangles their two responses for you as the room devolves into shouting and several Elders who look quite worried leave the room. A moment later all three of you are escorted out and the chambers are closed.`
-			`	Teeneep just shakes her head, pulls out a datapad, and quickly taps a few times. "Well, that was going to happen in an hour. No point waiting, though. Let's get dinner."`
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins, "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
```

Comment:
```suggestion
			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
```

Teeneep is threatening the council here, but she doesn't say explicitly what the council should do. Is it better to leave her implying that the Hai military should accompany the Unfettered, or make her say it outright? She doesn't say anything about it here. I might have missed something from an earlier mission that covers this for the player, though.

Reply:
The coordination with the Unfettered has to be much more than just a military escort. There's the matter of diplomacy with humans to prevent a war, and internal politics to get the Fettered Hai populace to accept it. I thought it would be best to avoid such a long discussion. Presumably, the council wouldn't get into such details unless they thought an agreement to work with Unfettered is likely. As such an agreement is almost unimaginable, I'd expect them to skip the talk unless the vote went in a deeply surprising direction.

Reply:
I'm hoping more people will review, so I'll wait to hear more before I make such a large change to the conversation.

Reply:
I remember now the original justification for the Unfettered help. The Fettered are beset on both sides by attackers, but Teeneep wants a deal where the Unfettered stop attacking the Fettered and guard the wormhole. 

That doesn't work anymore, now that the cease in intra-Hai hostilities is gone. I'll have to contrive another justification.

Reply:
I think the best solution is for Teeneep to demand the Elders allow the Unfettered to pass through Fettered territory and assist in defending the wormhole. Ultimately, that is what ends up happening, but only a few Unfettered special ops ships rather than a full alliance.

Reply:
Teeneep now insists on an agreement with the Unfettered that is also negotiated with the humans. That way, the player's continued cooperation isn't suspicious (as they may be intentionally starting a war otherwise). I've updated the player's possible responses to match the new Teeneep speech.